doc/api/constants.md

```diff
@@ -0,0 +1,1083 @@
+# Constants
+
+The `constants` module provides convenient access to commonly used constants for
+use with file access, crypto methods, error handling and more. No methods are
+exposed by the `constants` module itself.
+
+**Note:** Specific constants may not be available on every operating system.
+
+## Error Constants
+
+The following error constants are adapted directly from the Posix `errno`
+standards. See http://man7.org/linux/man-pages/man3/errno.3.html for more
```

Comment:
Yep, haven't gotten that far ;-) For most of these, the auto man page links should work (just need to go through and do the tedious work of adding those in. I'm open to suggestions on what to do with reconciling against errno.md.